### DISCUSSION TITLE
charm fails with "uncaught exception" after Starting surface creation

### DISCUSSION THREAD

**Discussion by sassecon** [simnibs/simnibs#186] [7.Creating head models]
Dear list, 

I am quite new to using SimNibs for head models: I am trying to create some individual head models from an open dataset. I only have T1 images and in Analyze format.
To start, I have converted Analyze to Nifti (using fslchfiletype), otherwise, I would get an error relating to the qform.
I then passed the Nifti images to Charm, which seemed to work (creating expected files up to the 'Starting surface creation' step.
At this point, I get the following error (from the charm_log.html).
I am a bit lost. I was wondering if someone has any idea what I did wrong.
(Using SimNIBS 4.0, charm init as by default (as I see in settings.ini), on a MAC Ventura 13.4)

Thank you in advance
Best
Sara

--------------------------------
CRITICAL: Uncaught exception
subprocess.CalledProcessError: Command '['/Users/saraassecondi/Applications/SimNIBS-4.0/simnibs_env/bin/python', '/Users/saraassecondi/Applications/SimNIBS-4.0/simnibs_env/lib/python3.9/site-packages/simnibs/segmentation/run_cat_multiprocessing.py', '--Ymf', '/Users/saraassecondi/Dropbox/workplace/University/Projects/2209-WM-SETUP/simulations/Data/OAS1_0002/m2m_OAS1_0002/surfaces/norm_image.nii.gz', '--Yleft_path', '/Users/saraassecondi/Dropbox/workplace/University/Projects/2209-WM-SETUP/simulations/Data/OAS1_0002/m2m_OAS1_0002/surfaces/hemi_mask.nii.gz', '--Ymaskhemis_path', '/Users/saraassecondi/Dropbox/workplace/University/Projects/2209-WM-SETUP/simulations/Data/OAS1_0002/m2m_OAS1_0002/surfaces/cereb_mask.nii.gz', '--surface_folder', '/Users/saraassecondi/Dropbox/workplace/University/Projects/2209-WM-SETUP/simulations/Data/OAS1_0002/m2m_OAS1_0002/surfaces', '--fsavgdir', '/Users/saraassecondi/Applications/SimNIBS-4.0/simnibs_env/lib/python3.9/site-packages/simnibs/resources/templates/fsaverage_surf', '--surf', 'lh', 'rh', '--pial', 'lh', 'rh', '--vdist', '1.0', '0.75', '--voxsize_pbt', '0.5', '0.25', '--voxsizeCS', '0.75', '0.5', '--th_initial', '0.714', '--no_intersect', 'True', '--nprocesses', '2']' returned non-zero exit status 1.

**Comment by sassecon** [comment]
Update: I think the problem may be in the conversion from Analyze to Nifti, giving wrong orientation/labels.
I am trying to solve this but haven't found any converter other than fslchfiletype.

**Reply by bjornsig** [reply]
Ok, please keep us informed if it is the file conversion that is the problem or it's something on our end.
